### DISCUSSION TITLE
Attributes don't render on Cloudflare Workers (but work locally)

### DISCUSSION THREAD

**Discussion by rodbs** [markdoc/markdoc#535] [Q&A] [answered]
### What happened?

Components built with Markdoc render properly in Remix on Cloudflare Pages using wrangler (locally). But when deploying to CF the attributes part of the component don't render.

### To reproduce

1. Clone this repo
 [https://github.com/rodbs/markdoc-cf]()
2. Run it locally (npm run dev). Components render properly
3. Deploy to Cloudflare Pages  (npm pages:deploy). Attributes side of the components don't render
[https://markdoc-cf.pages.dev/](url)

### Version

0.4.0

### Additional context
Local
![image](https://github.com/markdoc/markdoc/assets/4244067/57ac44ea-2380-4422-894c-39f2a771736e)
Deployed on CF
![image](https://github.com/markdoc/markdoc/assets/4244067/2b0d5354-2ab3-4d95-b5c3-20f1b8246069)

_No response_

**Comment by buremba** [comment]
I also run into the same issue

**Comment by maxblank-stripe** [comment, marked as answer]
hey @rodbs! The issue here isn't with the Cloudflare Workers, but with the production build of the app. The problem is that you're using `Callout.name` property [here](https://github.com/rodbs/markdoc-cf/blob/main/app/components/callout.tsx#L16), which resolves to `"Callout"` during development build, but is changed to `"pf"` in production build (screenshot from cloudflare pages logs below). In your example Markdoc is trying to render a component that doesn't exist.

![markdoc-demo](https://github.com/user-attachments/assets/33c0cf00-b88f-4358-b09b-220a66e9e7f8)

The easiest way to fix it would be to change `Callout.name` to be just a `"Callout"` string.

Another solution if you wanna keep using `Callout.name` would be to update [this line](https://github.com/rodbs/markdoc-cf/blob/main/app/components/markdown.tsx#L10) to pass the production name of the component to Markdoc as well:
```jsx
renderers.react(content, React, { components: {
  [Callout.name]: Callout,
  [Counter.name]: Counter,
} })
```
